common/normalize.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	"unicode"
+
+	"github.com/mtibben/confusables"
+	"golang.org/x/text/transform"
+	"golang.org/x/text/unicode/norm"
+)
+
+var transformer = transform.Chain(norm.NFD, transform.RemoveFunc(isMn), norm.NFC)
+
+func isMn(r rune) bool {
+	return unicode.Is(unicode.Mn, r)
+}
+
+// stupid name, cant think of a better one atm
+func FixText(content string) string {
```

Comment:
SanitizeText?

Reply:
That was actually the original name and then it was changed like 80 times lmao, ill change back to SanitizeText